zmon-common/src/main/java/org/zalando/zmon/domain/ExecutionStatus.java

```diff
@@ -17,12 +17,14 @@
     private final int queueSize;
     private final Set<Worker> workers;
     private final Set<Queue> queues;
+    private final ServiceLevelStatus.ServiceLevelStatusData serviceLevelStatus;
```

Comment:
It really not important, but you could explicitly import `ServiceLevelStatus.ServiceLevelStatusData` to avoid long lines through the code.

Reply:
changed it in `ZmonServiceImpl.java` as well :)